docs/cli.adoc

```diff
@@ -48,6 +48,19 @@ This command has two modes:
 Incompatible with *--lsp-server*.
 --
 
+*--diagnostic-hyperlinks*=_when_::
+  Control whether the errors are using escape sequences to show errors as hyperlinks or not.
```

Comment:
"show errors as hyperlinks" is weird phrasing IMO. Hyperlink is a verb (as well as a noun), so we can say "hyperlink errors".

But we're not really hyperlinking the errors; we're hyperlinking the error codes. So maybe write this:
```suggestion
  Control whether to hyperlink error codes or not.
```